.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 tmp
 *.log
-test/**/yarn.lock
\ No newline at end of file
+test/**/yarn.lock
```

Comment:
After reading this comment https://github.com/ember-template-lint/ember-template-lint/pull/768#discussion_r307409812 I'm going to remove this

Reply:
I've updated to remove the .gitignore addition